tlslite/keyexchange.py

```diff
@@ -84,6 +84,16 @@ def signServerKeyExchange(self, serverKeyExchange, sigHash=None):
 
         serverKeyExchange.signature = self.privateKey.sign(hashBytes)
 
+        if not serverKeyExchange.signature:
+            raise TLSIllegalParameterException("Empty signature")
```

Comment:
since this should in general should "never" happen, `TLSInternalError` would be a better choice